Backend/app/api/v1/text_endpoint.py

```diff
@@ -1,18 +1,41 @@
-from fastapi import APIRouter, Body
+from fastapi import APIRouter, HTTPException
+from pydantic import BaseModel
 
 router = APIRouter()
 
+class TextInput(BaseModel):
+    text: str
+
+class TextResponse(BaseModel):
+    text: str
+    data: str
+    code: int
+
```

Comment:
These Models should be extracted to two files, one for each model under a directory called "dto" or "entity".